mesonbuild/mesonlib.py

```diff
@@ -23,6 +23,7 @@
 import collections
 from enum import Enum
 from functools import lru_cache
+import operator
```

Comment:
__[Flake8]__ [F811] redefinition of unused 'operator' from line 22 ([link](https://sider.review/gh/repos/19784232/pulls/5312#issue-4002756))

<sub>posted by Sider</sub>